### ISSUE TITLE
Newlines are not allowed in summary

### ISSUE THREAD

**Issue by AlphaJack** [frederickjansen/polyline#16]
Hi, I got this warning while installing the package:

```
/usr/lib/python3.11/site-packages/setuptools/_core_metadata.py:157: SetuptoolsDeprecationWarning: Invalid config.
!!

        ********************************************************************************
        newlines are not allowed in `summary` and will break in the future
        ********************************************************************************

!!
  write_field('Summary', single_line(summary))

```

**Comment by frederickjansen** [comment]
Thanks, looks like a change in setuptools. This shouldn't be an issue for now but I'll fix this in the next release.

**Comment by frederickjansen** [comment]
Fixed in v2.0.2
